README.markdown

```diff
@@ -507,6 +507,20 @@ func reticulateSplines(spline: [Double], adjustmentFactor: Double,
 }
 ```
 
+When declaring a new function, we recommend using markup to create a richly formatted 'Quick Help' describing the function, it's paremeters and it's return object. This way, other users simply have to select the option key and select the function in any context to see the information about the function with it's intended use.
+
+To create a 'Quick Help' style decleration format your function like this:
```

Comment:
Mind adding the hotkey as well `OPT+CMD+/` in case people don't know it

Reply:
Oh my god I didn't know that!! That's awesome!